### DISCUSSION TITLE
有没有上传Base64图像的代码用例

### DISCUSSION THREAD

**Discussion by yang1158280920** [exadel-inc/CompreFace#689] [Q&A]
![image](https://user-images.githubusercontent.com/61954441/147085594-3aacc28d-f012-4d70-a417-f84abb4dd486.png)

![image](https://user-images.githubusercontent.com/61954441/147085831-d2670889-c96e-4f2d-8827-300077c63372.png)
![image](https://user-images.githubusercontent.com/61954441/147085918-bb674339-7b29-4f36-9cb9-306c2013253f.png)

I’m sure what I’m sending is in Base64 format. Why is it reported that I’m not sending in Base64 format?

**Comment by pospielov** [comment]
Try to remove the first part of base64 information
![image](https://user-images.githubusercontent.com/3736126/147126109-38533483-3442-422e-b768-9e5363770d43.png)

**Reply by yang1158280920** [reply]
This is indeed the reason, thank you. By the way, Merry Christmas
